File: 1010-calculoSimples.c
#include <stdio.h>

int main( void )
{
  int cod1, nPecas1, cod2, nPecas2;
  double valor1, valor2;

  scanf("%d %d %lf", &cod1, &nPecas1, &valor1 );
  scanf("%d %d %lf", &cod2, &nPecas2, &valor2 );

  printf("VALOR A PAGAR: R$ %.2f\n", nPecas1*valor1+nPecas2*valor2 );
  return( 0 );
}

File: 1010-calculoSimples.py
codigo1,qtd1,valor1=input().split(" ")
codigo2,qtd2,valor2=input().split(" ")

codigo1=int(codigo1)
qtd1=int(qtd1)
valor1=float(valor1)

codigo2=int(codigo2)
qtd2=int(qtd2)
valor2=float(valor2)

print("VALOR A PAGAR: R$ {:.2f}".format(qtd1*valor1+qtd2*valor2))
